pkg/innerring/config.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 // Various configuration paths.
-const (
-	cfgPathFSChain               = "morph"
+var (
```

Comment:
Shouldn't be needed.

Reply:
you did it many times below, what made you do so?

Reply:
I did this to change the name depending on which one was provided in the config.